### PR TITLE
DOC-528: Indexes for function folders (Pt. 1)

### PR DESCRIPTION
**Pull request by diana-macrometa**
This PR improves code block formatting and adds explanation to functions.

It is the first of several PRs that will do this.

@jamesm-macrometa, please merge this PR once SMEs approve it.

### CODE REVIEW

File: docs/cep/query-guide/functions/core/log.md [hunk 1]
```diff
@@ -6,65 +6,89 @@ Logs the message on the given priority with or without the processed event.
...
 ## Example 3
 
-    insert into BarStream
-    select *
-    from FooStream#log("DEBUG", "Sample Event :", true);
+```sql
+@info(name = 'query1')
+insert into BarStream
+select *
+from FooStream#log("DEBUG", "Sample Event :", true);
```
Comment (StoyanTc) [inline docs/cep/query-guide/functions/core/log.md]: @diana-macrometa - Shall we change the case here for the query?

File: docs/cep/query-guide/functions/core/log.md [hunk 2]
```diff
@@ -6,65 +6,89 @@ Logs the message on the given priority with or without the processed event.
...
 
-    insert into BarStream
-    select *
-    from FooStream#log("Event Arrived", false);
+```sql
+@info(name = 'query1')
+insert into BarStream
+select *
+from FooStream#log("Event Arrived", false);
+```
```
Comment (StoyanTc) [inline docs/cep/query-guide/functions/core/log.md]: @diana-macrometa - Shall we change the case here for the query?

File: docs/cep/query-guide/functions/core/log.md [hunk 3]
```diff
@@ -6,65 +6,89 @@ Logs the message on the given priority with or without the processed event.
...
 
-    insert into BarStream
-    select *
-    from FooStream#log("Sample Event :", true);
+```sql
+@info(name = 'query1')
+insert into BarStream
+select *
+from FooStream#log("Sample Event :", true);
+```
```
Comment (StoyanTc) [inline docs/cep/query-guide/functions/core/log.md]: @diana-macrometa - Shall we change the case here for the query?

File: docs/cep/query-guide/functions/core/max.md [hunk 4]
```diff
@@ -18,7 +18,7 @@ Returns the maximum value for all the events.
 
 ## Example
 
-```js
+```sql
     insert into barStream
     select max(temp) as maxTemp
     from fooStream WINDOW TUMBLING_TIME(10 sec);
```
Comment (StoyanTc) [inline docs/cep/query-guide/functions/core/max.md]: @diana-macrometa - Shall we change the case here for the query?

File: docs/cep/query-guide/functions/core/maxForever.md [hunk 5]
```diff
@@ -18,7 +18,7 @@ This is the attribute aggregator to store the maximum value for a given attribut
 
 ## Example
 
-```js
+```sql
     insert into outputStream
     select maxForever(temp) as max
     from inputStream;
```
Comment (StoyanTc) [inline docs/cep/query-guide/functions/core/maxForever.md]: @diana-macrometa - Shall we change the case here for the query?

File: docs/cep/query-guide/functions/core/maximum.md [hunk 6]
```diff
@@ -16,7 +16,7 @@ Returns the maximum value of the input parameters.
 
 ## Example
 
-```js
+```sql
 @info(name = 'query1') from inputStream
 insert into outputStream
 select maximum(price1, price2, price3) as max;
```
Comment (StoyanTc) [inline docs/cep/query-guide/functions/core/maximum.md]: @diana-macrometa - Shall we change the case here for the query?

File: docs/cep/query-guide/functions/core/min.md [hunk 7]
```diff
@@ -18,7 +18,7 @@ Returns the minimum value for all the events.
 
 ## Example
 
-```js
+```sql
 insert into outputStream
 select min(temp) as minTemp
 from inputStream;
```
Comment (StoyanTc) [inline docs/cep/query-guide/functions/core/min.md]: @diana-macrometa - Shall we change the case here for the query?

File: docs/cep/query-guide/functions/core/minForever.md [hunk 8]
```diff
@@ -18,7 +18,7 @@ This is the attribute aggregator to store the minimum value for a given attribut
 
 ## Example
 
-```js
+```sql
 insert into outputStream
 select minForever(temp) as max
 from inputStream;
```
Comment (StoyanTc) [inline docs/cep/query-guide/functions/core/minForever.md]: @diana-macrometa - Shall we change the case here for the query?

File: docs/cep/query-guide/functions/core/minimum.md [hunk 9]
```diff
@@ -16,7 +16,7 @@ Returns the minimum value of the input parameters.
 
 ## Example
 
-```js
+```sql
 @info(name = 'query1') from inputStream
 insert into outputStream
 select maximum(price1, price2, price3) as max;
```
Comment (StoyanTc) [inline docs/cep/query-guide/functions/core/minimum.md]: @diana-macrometa - Shall we change the case here for the query?

File: docs/cep/query-guide/functions/core/or.md [hunk 10]
```diff
@@ -18,7 +18,7 @@ Returns the results of OR operation for all the events.
 
 ## Example
 
-```js
+```sql
 insert into alertStream
 select or(isFraud) as isFraudTransaction
 from cscStream WINDOW TUMBLING_LENGTH(10);
```
Comment (StoyanTc) [inline docs/cep/query-guide/functions/core/or.md]: @diana-macrometa - Shall we change the case here for the query?

File: docs/cep/query-guide/functions/core/unionSet.md [hunk 11]
```diff
@@ -18,7 +18,7 @@ Union multiple sets. This attribute aggregator maintains a union of sets. The gi
 
 ## Example
 
-```js
+```sql
 insert into initStream
 select createSet(symbol) as initialSet
 from stockStream
```
Comment (StoyanTc) [inline docs/cep/query-guide/functions/core/unionSet.md]: @diana-macrometa - Shall we change the case here for the query?

File: docs/cep/query-guide/functions/core/uuid.md [hunk 12]
```diff
@@ -6,13 +6,13 @@ Generates a UUID (Universally Unique Identifier).
...
 <STRING> UUID()
 ```
 
 ## Example
 
-```js
+```sql
 insert into RoomTempStream
 select convert(roomNo, 'string') as roomNo, temp, UUID() as messageID
 from TempStream;
```
Comment (StoyanTc) [inline docs/cep/query-guide/functions/core/uuid.md]: @diana-macrometa - Shall we change the case here for the query?

**Review by StoyanTc — CHANGES_REQUESTED** [review]

**Comment by jamesm-macrometa** [comment]
Hi @StoyanTc - Diana is off for the week, so I'll be handling this in her stead.  🙂 

Can you please clarify what you mean by "change the case for the query"?  I assume you mean the casing i.e. upper-case or lower-case, but it's not clear to me from looking at the PR what specific change you are requesting.

**Comment by StoyanTc** [comment]
> Hi @StoyanTc - Diana is off for the week, so I'll be handling this in her stead. 🙂
> 
> Can you please clarify what you mean by "change the case for the query"? I assume you mean the casing i.e. upper-case or lower-case, but it's not clear to me from looking at the PR what specific change you are requesting.

Hi @jamesm-macrometa,

Apologies for my unclear and grammatically not correct comment.
The QL we use into the stream workers is case insensitive. However to make the queries more readable we put all keywords in uppercase. And this is already done everywhere except into the files I have added my comment.

E.g. 
```
insert into RoomTempStream
select convert(roomNo, 'string') as roomNo, temp, UUID() as messageID
from TempStream;
```
should become
```
INSERT INTO RoomTempStream
SELECT convert(roomNo, 'string') AS roomNo, temp, UUID() AS messageID
FROM TempStream;
```

**Comment by jamesm-macrometa** [comment]
Thanks, @StoyanTc !  I've updated the docs per your suggestion.  Can you please approve the changes so I can merge?

**Review by jamesm-macrometa — APPROVED** [review]

File: docs/cep/query-guide/functions/core/stdDev.md [hunk 13]
```diff
@@ -18,7 +18,7 @@ Returns the calculated standard deviation for all the events.
 
 ## Example
 
-```js
+```sql
 insert into outputStream
 select stddev(temp) as stdTemp
 from inputStream;
```
Comment (StoyanTc) [inline docs/cep/query-guide/functions/core/stdDev.md]: @jamesm-macrometa - Those lines have left in lower-case.

File: docs/cep/query-guide/functions/core/sum.md [hunk 14]
```diff
@@ -16,7 +16,7 @@ Returns the sum for all the events.
 
 ## Example
 
-```js
+```sql
 insert into outputStream
 select sum(volume) as sumOfVolume
 from inputStream;
```
Comment (StoyanTc) [inline docs/cep/query-guide/functions/core/sum.md]: @jamesm-macrometa - Those lines have left in lower-case.

**Review by StoyanTc — CHANGES_REQUESTED** [review]

**Comment by jamesm-macrometa** [comment]
Thanks again for the catches @StoyanTc !  Just updated per your last comments.  Please approve when ready 🙂

**Review by StoyanTc — APPROVED** [review]

**Pull request merged**